Add WebDriver conformance updates for 108 release notes (#22840)
mdn/content: files/en-us/mozilla/firefox/releases/108/index.md

#### WebDriver BiDi

> - Following a [change in the specification](https://github.com/w3c/webdriver-bidi/pull/259), log entry level `"warning"` was renamed to `"warn"` ([bug 1797115](https://bugzilla.mozilla.org/show_bug.cgi?id=1797115))

Reviewer: Just an FYI; there's a macro you may use in this repo for your convenience:

```markdown
<!-- markdown link -->
[bug 1797115](https://bugzilla.mozilla.org/show_bug.cgi?id=1797115)

<!-- shorthand / macro -->
{{bug(1797115)}}
```

Author: ah nice, I switched to the macro

> - When using `script.evaluate` and `script.callFunction` with a sandbox name equal to an empty string, the evaluation will be done using the default realm ({{bug(1793589)}})

Reviewer 2: I wonder if we should add `now`:

```suggestion
- When using `script.evaluate` and `script.callFunction` with a sandbox name equal to an empty string, the evaluation will now be done using the default realm ({{bug(1793589)}})
```

Author: sure :)

> - When using `script.evaluate` and `script.callFunction` without sandbox name, the evaluation will be done using the default realm ([bug 1793589](https://bugzilla.mozilla.org/show_bug.cgi?id=1793589))

Reviewer: ```suggestion
- When using `script.evaluate` and `script.callFunction` without a sandbox name, the evaluation will be done using the default realm ([bug 1793589](https://bugzilla.mozilla.org/show_bug.cgi?id=1793589))
```

Reviewer 3: To be accurate, the change is really about using an empty string as sandbox name. If you didn't provide a sandbox name, we were already using the default realm. But with `sandbox=""` we used to create sandbox named `""` (empty string). 

```suggestion
- When using `script.evaluate` and `script.callFunction` with a sandbox name equal to an empty string, the evaluation will be done using the default realm ([bug 1793589](https://bugzilla.mozilla.org/show_bug.cgi?id=1793589))
```

- Added support for the `browsingContext.domContentLoaded` event ([bug 1756610](https://bugzilla.mozilla.org/show_bug.cgi?id=1756610))

#### Marionette

> - Added support for `tiltX`, `tiltY` and `twist` on touch event ([bug 1793832](https://bugzilla.mozilla.org/show_bug.cgi?id=1793832))

Reviewer 2: ```suggestion
- Added support for the `tiltX`, `tiltY` and `twist` properties of pointer actions for `WebDriver:PerformActions` ([bug 1793832](https://bugzilla.mozilla.org/show_bug.cgi?id=1793832))
```

> - Fixed a bug where "Get Element Text" wasn't returning the element text for pretty-printed XML ([bug 1794099](https://bugzilla.mozilla.org/show_bug.cgi?id=1794099))

Reviewer: ```suggestion
- Fixed a bug where retrieving element text content for pretty-printed XML was failing ([bug 1794099](https://bugzilla.mozilla.org/show_bug.cgi?id=1794099))
```

> - Fixed a bug where "Get Element Text" wasn't returning the element text for pretty-printed XML ([bug 1794099](https://bugzilla.mozilla.org/show_bug.cgi?id=1794099))

Reviewer 2: Lets use the API endpoint of Marionette:

```suggestion
- Fixed a bug where `WebDriver:GetElementText` wasn't returning the element text for pretty-printed XML ([bug 1794099](https://bugzilla.mozilla.org/show_bug.cgi?id=1794099))
```

> - `HTMLDocument` are no longer serialized as a WebElement reference ([bug 1793920](https://bugzilla.mozilla.org/show_bug.cgi?id=1793920))

Reviewer 2: ```suggestion
- `HTMLDocument` is no longer serialized as a `WebElement` reference ([bug 1793920](https://bugzilla.mozilla.org/show_bug.cgi?id=1793920))
```
